packages/zeplin/src/index.ts

```diff
@@ -0,0 +1,73 @@
+import https from 'https'
+import { exit } from 'process'
+import { convertToCss } from './convertToCss'
+import { convertToTailwind } from './convertToTailwind'
+import { saveDocument } from './saveDocument'
+export type ProjectFramework = 'tailwind' | 'css'
+
+export async function main(
```

Comment:
`main` wordt twee keer ge-export, zou één keer genoeg zijn?

```suggestion
export default async function main(
```